the-blue-alliance-ios/View Controllers/Teams/TeamsViewController.swift

```diff
@@ -54,6 +54,8 @@ class TeamsViewController: TBATableViewController, Refreshable, Stateful, TeamsV
 
         // Used to make sure the UISearchBar stays in our root VC (this VC) when presented and doesn't overlay in push
         definesPresentationContext = true
+
+        updateInterface()
```

Comment:
Only calling this here is problematic because if data changes someplace else, we could go to this view and have it not be correct. We should look to update this count in `viewDidAppear` or something.